lib/stoplight/data_store/base.rb

```diff
@@ -11,26 +11,30 @@ def names
 
       # @param _light [Light]
       # @return [Array(Array<Failure>, String)]
-      def get_all(_light)
+      # @param window [Integer, nil]
+      def get_all(_light, window: nil)
```

Comment:
I usually do this for positional parameters, but it does not work for keyword args.  For instance, the generated documentation will be incorrect. It will say "you have to pass the _window option`  which is not true